bitnami/solr/values.yaml

```diff
@@ -0,0 +1,655 @@
+  ##
+  externalTrafficPolicy: Cluster
+
+## Bitnami Solr image version
+## ref: https://hub.docker.com/r/bitnami/solr/tags
+##
+image:
+  registry: docker.io
+  repository: bitnami/solr
+  tag: 8.7.0-debian-10-r66
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+## Configure extra options for liveness and readiness probes
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+##
+livenessProbe:
+  enabled: true
+  initialDelaySeconds: 30
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+readinessProbe:
+  enabled: true
+  initialDelaySeconds: 5
+  periodSeconds: 10
+  timeoutSeconds: 5
+  failureThreshold: 6
+  successThreshold: 1
+
+## Solr core name to be created
+##
+coreName: my-core
+## Enable Solr cloud
+##
+cloudEnabled: true
+## Enable cloud bootstrap. It will be performed from the node 0.
+##
+cloudBootstrap: true
+## Solr collection name
+##
+collection: my-collection
+## Solr number fo collection shards
+##
+collectionShards: 1
+## Solr number of collection replicas
+##
+collectionReplicas: 2
+## Port number for the Solr container
+##
+containerPort: 8983
+## Solr server directory
+##
+serverDirectory: "server"
+## Java memmory options
+##
+javaMem:
+## Java Heap options
+##
+heap:
+
+authentication:
+  ## Enabled Solr authentication
+  ##
+  enabled: true
+  ## Solr admin username
+  ##
+  adminUsername: admin
+  ## Solr admin password
+  ##
+  adminPassword:
+
+## Secret with Solr password
+##
+# existingSecret:
+
+## Container Command (set to default if not set).
+##
+command: []
+## Container args (set to default if not set).
+##
+args: []
+
+## Solr resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  limits: {}
+  #   cpu: 200m
+  #   memory: 256Mi
+  requests: {}
+  #   cpu: 200m
+  #   memory: 10Mi
+
+## Solr containers' Security Context
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-container
+##
+containerSecurityContext:
+  enabled: true
+  runAsUser: 1001
+  runAsNonRoot: true
```

Comment:
Missing sysctls, which is referenced in the README

Reply:
It is in the README because of a copy paste error. I will remove it from the README instead. It should not be needed here.